README.rst

```diff
@@ -60,13 +81,17 @@ contributions!
 
 See the `developer docs`_ and the `contribution docs`_ for more information.
 
+
 Help
 ====
 
 Looking for more help?
 
-- Read `the project documentation`_
-- Check out our `support channels`_
+- Read `the project documentation`_.
```

Comment:
I saw a PR recently (though I have lost track of it) that moved the documentation links into the main body of the README (for a different Crate project)

this link links to `/docs/`, which bounces you to the tutorials. at a minimum, we should probably change this to say something like "Try one of our beginner tutorials, how-to guides, or check out the reference manual" (adding links around "beginner tutorials, "how-to guides", and "reference manual")

alternatively, we could spin this out into a short subsection that links to the different docs projects one by one (perhaps using a list format). not sure

Reply:
ah of course. my bad. sorry! faulty memory hehe. thanks!